packages/adp-tooling/src/translations/adp-tooling.i18n.json

```diff
@@ -8,11 +8,35 @@
         "maxAgeTooltip": "(Optional) Enter the time in seconds it takes for a cached response of the service to expire.",
         "oDataAnnotationSourceURILabel": "OData Annotation Data Source URI",
         "oDataAnnotationSourceURITooltip": "Server side annotations have been detected for the selected OData service. If needed, enter new annotation data source URI  for the changed service",
-        "addAnnotationOdataSourceTooltip": "Select the annotation file from your workspace",
+        "oDataURIInfoLabel": "Have in mind that common OData Service path prefixes are '{{- v2Prefix}}' or '{{- v4Prefix}}'",
+        "isInSafeModeLabel": "Making manifest changes is not upgrade safe and the project will be set in advanced mode. Please continue if you are sure you want to have your project in advanced mode.",
+        "oDataURIV2InfoLabel": "Have in mind that you are trying to replace OData {{versionOne}} Service with OData {{versionTwo}} Service",
+
```

Comment:
These are not used anyware.

Reply:
These were introduced by a PR that you approved: https://github.com/SAP/open-ux-tools/pull/1925/files#diff-f4470a9e38ad8b522cd263b181ae9e2a3bf4e6edcb7463c010de5299f47ea1b6R11-R13